### ISSUE TITLE
GraphQL: Add query batching support

### ISSUE THREAD

**Issue by pushrbx** [Squidex/squidex#220]
See:
https://github.com/graphql-dotnet/graphql-dotnet/issues/398
https://github.com/graphql-dotnet/graphql-dotnet/issues/429
Example code:
https://github.com/graphql-dotnet/relay/blob/master/src/GraphQL.Relay/Http/RequestExecutor.cs

It would be nice to have support for query batching, which would allow to send multiple queries in one request.

**Comment by SebastianStehle** [comment]
Should be easy :) ... Basically you just have to use a list of query's right?

**Comment by pushrbx** [comment]
Yes, in case of angular, apollo-client sends an array of queries.

**Comment by SebastianStehle** [comment]
But I think the implementation is wrong. The definition says:

* Queries are executed in parallel
* Mutations are executed sequentially

Not a problem right now, but very, very soon: https://github.com/Squidex/squidex/pull/222

**Comment by pushrbx** [comment]
If queries are executed in parallel, what problems may occur in case of squidex? I fail to see the problem.

**Comment by SebastianStehle** [comment]
The problem are the mutations. We support them now and have to ensure that they are NOT executed in parallel ;)

**Comment by SebastianStehle** [comment]
I cannot find the definition anymore that mutations should be executed sequentially. Perhaps we can just ignore it for now and support it anyway

**Comment by pushrbx** [comment]
As #222 has been merged, I guess this can be closed? If the mutations bit is still a pending thing then I guess that will need a different issue.

**Comment by SebastianStehle** [comment]
I am confused

**Comment by pushrbx** [comment]
My bad, I was doing two things while reading through issues. (eatin) For some reason I thought PR #222 solves this issue, and I thought squidex already supports query batching.
